openshift/templates/wallet/wallet-build.param

```diff
@@ -0,0 +1,15 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/wallet/wallet-build.yaml
+#=========================================================
+NAME=wallet
+APP_NAME=issuer-kit
+APP_GROUP=issuer-kit-storage
+GIT_REPO_URL=https://github.com/bcgov/issuer-kit.git
+GIT_REF=master
+SOURCE_CONTEXT_DIR=wallet/config
+SOURCE_IMAGE_KIND=DockerImage
+SOURCE_IMAGE_NAME=registry.access.redhat.com/rhscl/postgresql-10-rhel7
```

Comment:
Should we be upgrading to postgres 12?  It's available in OCP4 as `"kind":"ImageStreamTag","name":"postgresql:12","namespace":"openshift"`

Reply:
Yes, that is a good idea. I logged a separate issue (#7) as this one was just to get things up and running based on the existing shared components.